spec/abci++/abci++_app_requirements_002_draft.md

```diff
@@ -629,9 +768,9 @@ the list should match the `AppHash` being verified against.
 When Tendermint connects to a peer, it sends two queries to the ABCI application
 using the following paths, with no additional data:
 
-- `/p2p/filter/addr/<IP:PORT>`, where `<IP:PORT>` denote the IP address and
+* `/p2p/filter/addr/<IP:PORT>`, where `<IP:PORT>` denote the IP address and
   the port of the connection
-- `p2p/filter/id/<ID>`, where `<ID>` is the peer node ID (ie. the
+* `p2p/filter/id/<ID>`, where `<ID>` is the peer node ID (ie. the
   pubkey.Address() for the peer's PubKey)
 
 If either of these queries return a non-zero ABCI code, Tendermint will refuse
```

Comment:
I haven't touched the crash recovery section, as I am focusing on that in my next PR